index.html

```diff
@@ -1222,6 +1222,25 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            The manifest is fetched and processed on every page load. When the
```

Comment:
I think we need to update the HTML spec to say this instead. I.e., it should say that, for regular web applications, the manifest is loaded when needed, but for "installed" web applications, the manifest is fetched on page load. 

Then here we can just say point to the HTML spec.

Reply:
Updated the html, but didn't find a way to reference it. Do you know how can I reference that?